code/modules/clothing/glasses/glasses.dm

```diff
@@ -149,6 +149,19 @@ var/list/science_goggles_wearers = list()
 	icon_state = "eyepatch"
 	item_state = "eyepatch"
 	min_harm_label = 0
+	var/flipped = FALSE
+
+/obj/item/clothing/glasses/eyepatch/attack_self()
+	if(!flipped)
+		flipped = TRUE
+		icon_state = "eyepatch2"
+		item_state = "eyepatch2"
+		to_chat(usr, "You flip the [src] to your left eye.")
```

Comment:
This will say "You flip the the eyepatch to your left eye"
Change it to `\the [src]`

Reply:
👍